docs/api/data/contracts/agent_registry/contract.md

```diff
@@ -1,71 +0,0 @@
-<a id="autonomy.data.contracts.agent_registry.contract"></a>
```

Comment:
Looks like a merge error? Suggest getting latest main merged and running generators again